### ISSUE TITLE
Port Metrics SDK to OpenTelemetry

### ISSUE THREAD

**Issue by felixnext** [felixnext/python-functown#16]
It seems that OpenCensus and OpenTrace have merged into https://opentelemetry.io/. Make sure that we update the SDK used to ensure long-term support.

See getting started guide: https://opentelemetry.io/docs/instrumentation/python/getting-started/

Azure SDK Tutorial: https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/telemetry-application-insights?pivots=programming-language-python

Depends on #10

**Comment by felixnext** [comment]
Putting this on the back burner for now, since the OpenTelemetry exporter for Python are still in preview and do not support logging (which would be a downgrade if not supported) and live metrics (which would be an update if supported).

See docs here: https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=python

Should monitor it and make the switch once it moves out of preview
